third_party/pcsc-lite/naclport/server_clients_management/src/client-handler-unittest.js

```diff
@@ -69,5 +70,35 @@ goog.exportSymbol('testClientHandler', {
 
     handler.dispose();
   },
+
+  'testClientChannelDisposed': function() {
+    const handler = new ClientHandler(
+        mockServerMessageChannel, goog.Promise.resolve(),
+        mockClientMessageChannel, EXTENSION_ID_A);
+    mockClientMessageChannel.dispose();
```

Comment:
nit: Maybe extract this into a separate function? This `.dispose()` together with the `disposeInternal` hack.